controllers/storagecluster/storageclasses.go

```diff
@@ -407,6 +407,7 @@ func newEncryptedCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageC
 	encryptedStorageClassConfig.storageClass.ObjectMeta.Name = generateNameForEncryptedCephBlockPoolSC(initData)
 	// adding a annotation to support smart cloning across namespace for encrypted volume
 	encryptedStorageClassConfig.storageClass.ObjectMeta.Annotations["cdi.kubevirt.io/clone-strategy"] = "copy"
+	encryptedStorageClassConfig.storageClass.ObjectMeta.Annotations["reclaimspace.csiaddons.openshift.io/schedule"] = "@weekly"
```

Comment:
@iamniting can you please add this to all rbd storageclasses ocs operator is creating?